### PR TITLE
[DOCFIX] Update cn version of Azure-Data-Lake doc

### PR DESCRIPTION
**Pull request by zhezhidashi**
### What changes are proposed in this pull request?

[DOCFIX] Update cn version of Azure-Data-Lake doc

### Why are the changes needed?

The Chinese ufs/Azure-Data-Lake doc has been created.

### Does this PR introduce any user facing changes?

Developers can get to know Alluxio in Chinese easily.

### CODE REVIEW

File: docs/cn/ufs/Azure-Data-Lake.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+title: Azure 数据湖存储
+nickname: Azure 数据湖存储
+group: Storage Integrations
+priority: 2
+---
+
+* Table of Contents
+{:toc}
+
+本指南介绍如何配置Alluxio，使其与底层存储系统 [Azure Data Lake Storage Gen1](https://docs.microsoft.com/en-in/azure/data-lake-store/data-lake-store-overview) 一起运行。
```
Comment (dbw9580) [inline docs/cn/ufs/Azure-Data-Lake.md:12]: how about linking to the Chinese doc for Azure Data Lake https://learn.microsoft.com/zh-cn/azure/data-lake-store/data-lake-store-overview

**Review by dbw9580 — APPROVED** [review]
LGTM

**Comment by LuQQiu** [comment]
@zhezhidashi can you signed the CLA form [here](https://www.alluxio.io/app/uploads/2022/07/Contribution-License-Agreement-2022-Template.pdf), so that we are able to move forward with this PR?

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**